build.gradle

```diff
@@ -231,10 +231,10 @@ patchPluginXml {
         <li> When the contents of the Liberty tool window are too large to fit in its window, a scroll bar appears.
         <li> Helpful advice is displayed when no project appears in the Liberty tool window.
         <li> Added the Language Servers tool window, which you can use when you need technical support.
-        <li> Updated the <a href="https://github.com/OpenLiberty/liberty-language-server">Liberty Config Language Server</a> version to 2.1.2. For more information about the changes for version 2.1.2, refer to the following release notes:
+        <li> Updated the <a href="https://github.com/OpenLiberty/liberty-language-server">Liberty Config Language Server</a> version to 2.2. For more information about the changes for version 2.2, refer to the following release notes:
```

Comment:
You can't change this here. These are the release notes for 24.0.3. Ask Trevor how to handle release notes for 24.0.12 (or whatever).

Reply:
Sure @turkeylurkey. Reverted the modifications in change notes